aspnetcore/toc.yml

```diff
@@ -1245,6 +1245,8 @@ items:
             uid: security/authentication/index
           - name: Introduction to Identity
             uid: security/authentication/identity
+          - name: Authentication and authorization in minimal APIs
```

Comment:
Can we put this under the minimal APIs heading instead?

Reply:
> Can we put this under the minimal APIs heading instead?

Sure, but it's a security topic and we don't do that for any other security topics. That's why I plastered all the Min API topics with a link the article. cc @tdykstra